pkg/store/project_head/project.go

```diff
@@ -0,0 +1,22 @@
+package projecthead
```

Comment:
the folder name should be `pkg/store/projecthead

Reply:
updated